### PR TITLE
Refactored code (added appropriate naming and styling)

### PR DESCRIPTION
**Pull request by maksym-studenets**
Refactored some code
Very important fixes.

### CODE REVIEW

File: SeaBattle/SeaBattle/GameBoard.cs [hunk 1]
```diff
@@ -128,31 +123,35 @@ private bool CanAddShip(Ship s, int y, int x, Orientation orientation)
...
+                }
+            }
         }
 
 
         private void prepareShips()
         {
             int i;
-            for (i = 0; i < Settings.fourDeckersCount; i++)
+            for (i = 0; i < Settings.fourDeckersCount; i++) {
```
Comment (Haidukov) [inline SeaBattle/SeaBattle/GameBoard.cs]: It is C# not Java - move curly brackets to the next line

Comment (Haidukov) [reply SeaBattle/SeaBattle/GameBoard.cs]: P.S. Check all pr

File: SeaBattle/SeaBattle/Settings.cs [hunk 2]
```diff
@@ -6,10 +6,10 @@ namespace SeaBattle
...
     class Settings
     {
-        public static int boardSize = 10;
-        public static int oneDeckersCount = 4;
-        public static int twoDeckersCount = 3;
-        public static int threeDeckersCount = 2;
-        public static int fourDeckersCount = 1;
+        public static const int boardSize = 10;
+        public static const int oneDeckersCount = 4;
+        public static const int twoDeckersCount = 3;
```
Comment (Haidukov) [inline SeaBattle/SeaBattle/Settings.cs]: Make constants UPPER CASE with underscore

File: SeaBattle/SeaBattle/Ship.cs [hunk 3]
```diff
@@ -28,8 +28,9 @@ public Ship(int length)
 
        public bool addDeck(Content content)
         {
-            if (decks.Count > length)
+            if (decks.Count > length) {
```
Comment (Haidukov) [inline SeaBattle/SeaBattle/Ship.cs]: I think we can leave if body in one line with condition

**Comment by Haidukov** [comment]
Do you think that the fixes are really "very" important? Can I ask you to give more appropriate name for prs?

File: SeaBattle/SeaBattle/Ship.cs (hunk 3 above)
Comment (maksym-studenets) [reply SeaBattle/SeaBattle/Ship.cs]: I am very happy that you are able to think but it is important ALWAYS to follow the convention. The convention says that parentheses should be used for if and for and similar blocks. You shouldn't utilize selective approach to following conventions. I am using the same approach in all cases. So, SCREW IT UP.

File: SeaBattle/SeaBattle/GameBoard.cs (hunk 1 above)
Comment (maksym-studenets) [reply SeaBattle/SeaBattle/GameBoard.cs]: Of course, I will. Any whim for your money

**Comment by maksym-studenets** [comment]
@Haidukov Of course they are important. This PR's mission is to clean the codebase from shitty code and I am actively doing it.

P.S. You can ASK, but I do not guarantee that I will fulfill that request

**Pull request merged**